components/CollectorPkgInstallInstructions.tsx

```diff
@@ -54,19 +54,21 @@ const CollectorDistroInstallInstructions: React.FunctionComponent<Pick<Props, 'k
   switch (kind) {
     case 'yum':
       installOpts = [
+        [ 'el9', 'el/9', 'RHEL / Rocky / OL 9' ],
         [ 'el8', 'el/8', 'RHEL / Rocky / OL 8' ],
         [ 'el7', 'el/7', 'RHEL / CentOS 7' ],
+        [ 'al2023', 'el/9', 'Amazon Linux 2023' ],
```

Comment:
With our install script, we pick el/9 for amazon linux 2023: https://github.com/pganalyze/collector/blob/94bef78c0dc84b049d963c22692094482d085269/contrib/install.sh#L62-L65